assets/collections-featured.css

```
@@ -0,0 +1,109 @@
+.collections {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px 33px;
+  gap: 24px;
+  margin: 20px 0;
+}
+
+.collection {
+  display: flex;
+  border-radius: 12px;
+  width: 100%;
+  position: relative;
+}
+
+.collection__info {
+  padding: 38px 14px 38px 30px;
+}
+
+.collection__info-subtitle {
+  color: white;
+  opacity: 80%;
+  font-weight: 500;
+  font-size: 18px;
+  letter-spacing: 1%;
+  margin-bottom: 20px;
+}
+
+.collection__info-title {
+  color: white;
+  font-weight: 700;
+  font-size: 45px;
+  letter-spacing: -1%;
+  margin-bottom: 55px;
+}
+
+.collection-featured__img {
+  display: none;
+}
+
+.collection__info-link {
+  text-transform: uppercase;
+  color: white;
+  font-size: 14px;
+  letter-spacing: 6%;
+  text-decoration: none;
+  border-radius: 8px;
+  padding: 20px 82px;
+  background-color: rgba(255, 255, 255, 0.3);
+  cursor: pointer;
+  box-shadow: 0px 4px 12px 0px rgba(209, 77, 65, 0.1);
+
+}
+
+.collection__info-title--first-element {
+  background: linear-gradient(to right, rgba(12, 234, 207, 1), rgba(11, 161, 143, 1));
+}
+
+.collection__info-title--second-element {
+  background: linear-gradient(to right, rgba(247, 203, 32, 1), rgba(248, 169, 4, 1));
+}
```

You should allow the admin to choose the background color, either with a [color](https://shopify.dev/docs/themes/architecture/settings/input-settings#color) or [color_background](https://shopify.dev/docs/themes/architecture/settings/input-settings#color_background) setting. You can set a default color here and allow the user to change it in the block settings by setting it inline or using a style block.

```suggestion
.collection {
  background: rgb(247, 203, 32);
}
```